testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oidc/LightWeightAccessTokenTest.java

```diff
@@ -230,24 +227,84 @@ public void exchangeTest() throws Exception {
             oauth.nonce("123456");
             oauth.scope("address");
 
-            oauth.clientId("test-app");
-            OAuthClient.AccessTokenResponse response = browserLogin("password", "test-user@localhost", "password").tokenResponse;
+            oauth.clientId(TEST_CLIENT);
+            OAuthClient.AccessTokenResponse response = browserLogin(TEST_CLIENT_SECRET, TEST_USER_NAME, TEST_USER_PASSWORD).tokenResponse;
             String accessToken = response.getAccessToken();
             System.out.println("accessToken:" + accessToken);
             assertAccessToken(oauth.verifyToken(accessToken), true, false);
-            response = oauth.doTokenExchange(TEST, accessToken, null, "test-app", "password");
+            response = oauth.doTokenExchange(TEST, accessToken, null, TEST_CLIENT, TEST_CLIENT_SECRET);
             String exchangedTokenString = response.getAccessToken();
             System.out.println("exchangedTokenString:" + exchangedTokenString);
 
-            oauth.clientId("resource-server");
-            String tokenResponse = oauth.introspectAccessTokenWithClientCredential("resource-server", "password", exchangedTokenString);
+            oauth.clientId(RESOURCE_SERVER_CLIENT_ID);
+            String tokenResponse = oauth.introspectAccessTokenWithClientCredential(RESOURCE_SERVER_CLIENT_ID, RESOURCE_SERVER_CLIENT_PASSWORD, exchangedTokenString);
             System.out.println("tokenResponse:" + tokenResponse);
             assertTokenIntrospectionResponse(JsonSerialization.readValue(tokenResponse, AccessToken.class), true, true, true);
         } finally {
             deleteProtocolMappers(protocolMappers);
         }
     }
 
+    @Test
+    public void testPolicyLightWeightFalseTest() throws Exception {
+        setUseLightweightAccessTokenExecutor();
+        ProtocolMappersResource protocolMappers = setProtocolMappers(true, true, false, true);
+        try {
+            oauth.nonce("123456");
+            oauth.scope("address");
+
+            oauth.clientId(TEST_CLIENT);
+
+            OAuthClient.AuthorizationEndpointResponse authsEndpointResponse = oauth.doLogin(TEST_USER_NAME, TEST_USER_PASSWORD);
+            OAuthClient.AccessTokenResponse tokenResponse = oauth.doAccessTokenRequest(authsEndpointResponse.getCode(), TEST_CLIENT_SECRET);
+            String accessToken = tokenResponse.getAccessToken();
+            assertAccessToken(oauth.verifyToken(accessToken), true, false);
+            System.out.println("access token:" + accessToken);
```

Comment:
How about replacing `System.out.println` with other logging method? It seems that Keycloak tends not to use `System.out.println`. Could you check other lines using `System.out.println` in this test class?

Reply:
Changed to use logger.debug.